svarut-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/svarut/SvarUtService.java

```diff
@@ -0,0 +1,21 @@
+package no.difi.meldingsutveksling.serviceregistry.svarut;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SvarUtService {
+
+    private SvarUtClient svarUtClient;
+
+    @Autowired
+    SvarUtService(SvarUtClient svarUtClient) {
+        this.svarUtClient = svarUtClient;
+    }
+
+    public boolean getSvarUtAdressering(String orgnr) {
+        RetrieveMottakerSystemForOrgnr request = RetrieveMottakerSystemForOrgnr.builder().withOrganisasjonsnr(orgnr).build();
+        RetrieveMottakerSystemForOrgnrResponse response = svarUtClient.retrieveMottakerSystemForOrgnr(request);
+        return response.getReturn().size() > 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1155)